HVCAEN/iocBoot/iocHVCAENSIM-IOC-01/config.xml

```diff
@@ -1,6 +1,9 @@
 <?xml version="1.0" ?>
 <ioc_config xmlns:xi="http://www.w3.org/2001/XInclude">
-<config_part>
-<ioc_desc>Simulated CAEN high voltage power supply</ioc_desc>
-</config_part>
+	<config_part>
+		<ioc_desc>Simulated CAEN high voltage power supply</ioc_desc>
+		<macros>
+			<macro name ="IFREADONLY" pattern="^ |#$" description="Whether device is readonly. Set to # for readonly, default is ' ' (just spacebar), for read and write" />
+		</macros>
+	</config_part>
```

Comment:
This isn't a very user-friendly macro to set, even with the additional description/explanation. As per comment below maybe just pass the ASG directly here as `DEFAULT`/`READONLY` as this is clearer.